Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Simple usage with a mounted data directory:
```

Comment:
Is this docker file also going to install a geth full node?

Reply:
it will not. configuration file must have the endpoint. 

Could include a docker-compose that deploys a geth node